.gitignore

```diff
@@ -0,0 +1,29 @@
+### IntelliJ IDEA ###
```

Comment:
please add .idea directory in git ignore file, as these directories are created by intellij and should not be part of codebase